src/core_plugins/kibana/common/tutorials/filebeat_instructions.js

```diff
@@ -165,24 +273,42 @@ export const FILEBEAT_INSTRUCTIONS = {
         'setup.kibana:',
         '  host: "<kibana_url>"'
       ],
-      textPost: 'Where `<password>` is the password of the `elastic` user, ' +
-                '`<es_url>` is the URL of Elasticsearch, and `<kibana_url>` is the URL of Kibana.'
+      textPost: i18n.translate('kbn.common.tutorials.filebeatInstructions.config.windowsTextPost', {
+        // eslint-disable-next-line
+        defaultMessage: 'Where {passwordTemplate} is the password of the `elastic` user, {esUrlTemplate} is the URL of Elasticsearch,\
+          and {kibanaUrlTemplate} is the URL of Kibana.',
+        values: {
+          passwordTemplate: '`<password>`',
+          esUrlTemplate: '`<es_url>`',
+          kibanaUrlTemplate: '`<kibana_url>`',
+        },
+      }),
     }
   },
   PLUGINS: {
     GEOIP_AND_UA: {
-      title: 'Install Elasticsearch GeoIP and user agent plugins',
-      textPre: 'This module requires two Elasticsearch plugins that are not ' +
-               'installed by default.\n\nFrom the Elasticsearch installation folder, run:',
+      title: i18n.translate('kbn.common.tutorials.filebeatInstructions.plugins.geoipUiTitle', {
```

Comment:
Looks like this is supposed to be `kbn.common.tutorials.filebeatInstructions.plugins.geoipUaTitle`, not `UiTitle`